src/Tracker/index.tsx

```diff
@@ -25,6 +25,7 @@ const integratorTrackedEvents = new Map<
   UserAnalyticsEventNames
 >([
   ['screen_welcome', 'WELCOME'],
+  ['screen_userConsent', 'USER_CONSENT'],
```

Comment:
Why is the event name format different for this screen?

Reply:
`userConsent` is the name of the step, so it becomes `screen_userConsent`. We do the same on `screen_crossDevice`